react/features/base/react/components/styles.web.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
I'm not sure about this pattern. We only seem to use it in one feature (JWT), and IMHO it makes reading the code more difficult because the reader may think styling is done in JS, but in web it's not. Can you abstract the components in such a way `styles.js` is only imported on the native side of things?

Reply:
- [ ] remove `styles.web.js` and `styles.native.js`